### ISSUE TITLE
000408 seems to be public but metadata in DLP says "dandi:EmbargoedAccess" 

### ISSUE THREAD

**Issue by yarikoptic** [dandi/dandi-archive#2093]
https://dandiarchive.org/dandiset/000408/draft says that it is embargoed, offers to unembargo (when I am logged in and I am admin), for others it just shows the dandiset and allows to browse/view data:

![image](https://github.com/user-attachments/assets/c3fcd914-1ffb-4a8e-b394-1ccdccf67eb0)

shouldn't it be hidden? was there a problem during unembargo which just didn't change metadata but otherwise opened it?

edit note: originally pointed to wrong dandiset https://dandiarchive.org/dandiset/000344 which was the one I compared against -- my test embargoed, which is "fully" embargoed and not shown as @kabilar demonstrated [below](https://github.com/dandi/dandi-archive/issues/2093#issuecomment-2523903346)

**Comment by kabilar** [comment]
Hi @yarikoptic, just to clarify, is this in regards to Dandiset 408 or 344?  The link goes to 344 but the screenshot is 408.

**Comment by kabilar** [comment]
When I am not logged in, 344 does have this screen:

![image](https://github.com/user-attachments/assets/3ed94f73-d5c6-44f3-a3b8-9341ab002486)

**Comment by kabilar** [comment]
For 408, I bet that this is just a bug in the `Access information` metadata as it seems to be unembargoed.

**Comment by yarikoptic** [comment]
@kabilar thank you! I fixed and extended description.  Indeed it might be just some residual remainder of "experimental manual unembargoings" and might just need metadata fix?

**Comment by yarikoptic** [comment]
meditor seems to not expose/allow to edit that field. So @jjnesbitt @waxlamp what is the best way to fix metadata here? should we arrange "pair DB" session?

**Comment by jjnesbitt** [comment]
It seems this is an uncaught remnant of #1954. In that PR, we update how `access` is treated, to ensure this value matches the dandiset embargo status. However, that only affects changes going forward, and any dandisets that haven't been updated since that changed wouldn't see that new value. Since that PR was merged on June 18th, and this dandiset was last updated May 8th, it would seem this is the culprit. 

I thought when I merged that PR I had gone through every dandiset and called `.save`, to initiate this setting of the `access` field. For one reason or another, this dandiset didn't get caught in that. However, querying the DB, I can confirm that this dandiset is the only one with this issue, so it should be straight forward to fix.

I can take care of this soon.
